draft-documents/a-roadmap-for-ssi.md

```diff
@@ -1,346 +1,166 @@
+* The release of a handful of reference applications that provide a down payment on the promise of decentralized identity
+* Formalized plans by year end 2018 for the foundational SSI specifications
+	* W3C DID
+	* W3C VC
+	* Oasis DKMS
+* Go to Market Resources to support developers by year end 2018
+	* Glossary of Terms
+	* Convergence of technical primers (tutorials) into a single  getting started kit
+	* Common baseline talking points for SSI advocates
+	* High impact videos and other demo recordings (i.e.: RWOT)
+	* Portfolio of industry specific demonstrations
+* Foundational catalysts for adoption
+	* Creation of a realistic technology roadmap
+	* Adoption of a formal agile project management approach to drive the roadmap
+	* Recruitment of foundational partners for a social media platform
+		* Operational recruitment of key technologists from Slack, Telegram and RocketChat
+		* Ideological recruitment of influential leaders (i.e: Reid Hoffman, Fred Wilson, etc)
+		* Get some traction on meaningful support from GitHub and Wikipedia
+
+## Reality Check
+
+We are a community of very passionate and bright individuals who share a common vision about self-sovereign identity (SSI). We have as an open community established the foundational building blocks for bringing the SSI vision to fruition. We witnessing a perfect storm whereby the juxtaposition of the fear of identity related hacks and thief are pervasive; decentralized identity and key management is now possible because of enabling technologies like blockchain; and governments and business are realizes the risks and liabilities associated with centralized identity systems. 
+
+### Preventing Technical Debit
+
+This perfect storm arrives at a time when non-SSI alternatives are also being considered at scale. If these alternatives gain traction inside key adoption communities like DMVs, which are historically slow to change, our failure to focus on SSI adoption will indirectly allow for new alternatives to get entrenched thereby making our adoption challenge more difficult. For example, one of our core worldwide identity instruments, the digital driver license, is being considered by national and state governments worldwide. Unfortunately, the new mobile driver license (mDL) initiative that is being considered by [ISO](https://www.iso.org/standard/41920.html) will yield technical debit within our society that will make the adoption of SSI even more difficult.  The time is now to get focused on an clear and achievable rollout roadmap for SSI technology.    
+ 
+### Resetting Priorities
+An honest assessment of the state of our SSI initiative will yield the reality that we are “not ready for prime time”. If we can get the community to agree on short and long term priorities, we have a very good chance to succeed and prevent unnecessary adoption hurdles. So instead of worrying about when/if DNA chips may be used to track humans, we should focus on:  
+
```

Comment:
Edit "So instead of worrying about when/if DNA chips may be used to track humans, we should focus on:" to "So instead of worrying about when/if DNA chips may be used to track humans (which is really a fun and thought provoking topic, and we encourage discussion), we *need* to focus on:

Reply:
Made changed and submitted another pull request.